composer.json

```diff
@@ -7,80 +7,83 @@
         "OSL-3.0",
         "AFL-3.0"
     ],
+    "config": {
+        "sort-packages": true
```

Comment:
Here's what I ran after adding this section to trigger sorting packages (without updating any dependencies):

```
$ composer require composer/composer:1.4.1
$ composer require --dev squizlabs/php_codesniffer:3.0.1
```